packages/relay-runtime/store/__tests__/RelayPublishQueue-test.js

```diff
@@ -1334,11 +1348,14 @@ describe('RelayPublishQueue', () => {
       );
       // Query payload sets name to 'zuck'
       queue.commitPayload(createOperationSelector(NameQuery, {id: '4'}), {
-        me: {
```

Comment:
this wasn't even a valid source. `createOperationSelector` supplied `id: 4` but all that was in the store was `me`.
now you can't notify the store with garbage payloads :smile: